k8s/client.go

```diff
@@ -634,8 +634,27 @@ func (client *Client) FindDeploymentByLabel(ctx context.Context, label string) (
 	return client.AppsV1().Deployments(client.Namespace).List(ctx, metav1.ListOptions{LabelSelector: label})
 }
 
-func (client *Client) DeployCoreOperatorSync(ctx context.Context, coreCloudURL, fromCloudImage, toCloudImage string, metricsPort string, memoryLimit string, annotations string, tolerations string, skipServiceCreation, noTLSVerify bool, cloudProxy, httpProxy, httpsProxy, noProxy string, coreInstance cloud.CreatedCoreInstance, serviceAccount string) (*appsv1.Deployment, error) {
-	podTolerations, err := validateTolerations(tolerations)
+type DeployCoreOperatorSyncParams struct {
```

Comment:
```suggestion
type DeployCoreOperatorSync struct {
```

The name should not collide since the other one is a method ;)